x/swap/types/errors.go

```diff
@@ -4,8 +4,7 @@ import (
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-// DONTCOVER
-
 var (
-	ErrCustom = sdkerrors.Register(ModuleName, 2, "")
+	ErrNotAllowed     = sdkerrors.Register(ModuleName, 1, "not allowed")
+	ErrNotImplemented = sdkerrors.Register(ModuleName, 99, "not implemented")
```

Comment:
```suggestion
	ErrNotImplemented = sdkerrors.Register(ModuleName, 3, "not implemented")
```
Our standard so far has been to order error codes in sequential order

Reply:
👍